### DISCUSSION TITLE
What's the best place to put the google fonts link? (_app, _document, or global styles)

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#24609] [Help]
Hi guys,
What's the best place to put the google fonts link?  _app, _document, or global styles?

I usually put it in _app.tsx or in my global style imported in _app... But today a friend asked me this question, and I'm not sure this is the best practice...

Thanks so much!

**Comment by gomflo** [comment]
Maybe this could help? https://github.com/vercel/next.js/discussions/13408#discussioncomment-18763

**Reply by gomflo** [reply]
There's actually a branch for `Font Optimization` documentation and the way they recommend its using `_document.js`

https://github.com/vercel/next.js/pull/24572/files/9b2eaa7dd53fd753bc1ca3a6f8fb111995d4e924

```js
// or to your entire application with a [Custom `Document`](/docs/advanced-features/custom-document.md).
// pages/_document.js
import Document, { Html, Head, Main, NextScript } from 'next/document'
class MyDocument extends Document {
  render() {
    return (
      <Html>
        <Head>
          <link
            href="https://fonts.googleapis.com/css2?family=Inter"
            rel="stylesheet"
          />
        </Head>
        <body>
          <Main />
          <NextScript />
        </body>
      </Html>
    )
  }
}
export default MyDocument
```

**Reply by rcapdepaula** [reply]
I'm using the official example of Vercel to set up Styled Components `{ ServerStyleSheet} ` in _document.js

[_document.js](https://github.com/vercel/next.js/blob/canary/examples/with-styled-components/pages/_document.js)

How do I set up Google Fonts to reach my entire project?  Some good soul can help me?
